drfc_manager/transformers/general.py

```
@@ -44,7 +49,11 @@
     try:
         storage_manager.copy_object(source_object_name, dest_object_name)
     except Exception as e:
-        raise BaseExceptionTransformers(f"Failed to copy S3 object from {source_object_name} to {dest_object_name}", e)
+        raise BaseExceptionTransformers(
+            f"Failed to copy S3 object from {source_object_name} to {dest_object_name}",
+            e,
+        )
```

Ruff (0.11.9)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)